reana_workflow_controller/workflow_run_managers/__init__.py

```diff
@@ -27,15 +28,20 @@ class WorkflowRunManager():
     engine_mapping = {
         'cwl': {'image': 'reanahub/reana-workflow-engine-cwl:{}'.format(
             WORKFLOW_ENGINE_VERSION),
-                'command': ['sleep', '1000'],
+                'command': "sleep 1000",
                 'environment_variables': common_env_variables},
         'yadage': {'image': 'reanahub/reana-workflow-engine-yadage:{}'.format(
             WORKFLOW_ENGINE_VERSION),
-                   'command': ['sleep', '1000'],
-                   'environment_variables': common_env_variables},
+                'command': "sleep 1000",
+                'environment_variables': common_env_variables},
         'serial': {'image': 'reanahub/reana-workflow-engine-serial:{}'.format(
             WORKFLOW_ENGINE_VERSION),
-                   'command': ['sleep', '1000'],
+                   'command': ("run-serial-workflow "
+                               "--workflow-uuid {id} "
+                               "--workflow-workspace {workspace} "
+                               "--workflow-json '{json}' "
+                               "--workflow-parameters '{parameters}' "
+                               "--operational-options {options} "),
```

Comment:
Quote `options` like `parameters`?

(we can revise later whether this is safe regarding escaping/unescaping... there are ways to encode/decode inputs for example)